packages/db/src/meta/pouch/adapters/fs.ts

```diff
@@ -15,10 +15,25 @@ import Config from "@truffle/config";
 export interface DatabasesSettings {
   directory: string;
 }
+type UserConfigDbSettings = {
```

Comment:
missing blank line between top-level identifiers:

```suggestion
}

type UserConfigDbSettings = {
```